testsuite/features/secondary/min_rhlike_openscap_audit.feature

```diff
@@ -38,11 +52,13 @@ Feature: OpenSCAP audit of Red Hat-like Salt minion
     When I follow "Audit" in the content area
     And I follow "xccdf_org.open-scap_testresult_standard"
     Then I should see a "Details of XCCDF Scan" text
-    And I should see a "RHEL-7" text
+    And I should see a "RHEL-8" text
     And I should see a "XCCDF Rule Results" text
-    When I enter "pass" as the filtered XCCDF result type
-    And I click on the filter button
-    Then I should see a "rpm_verify_permissions" link
+    # Workaround for the missing Rocky 8 openSCAP profile
+    # See https://github.com/SUSE/spacewalk/issues/19259
+    # When I enter "pass" as the filtered XCCDF result type
+    # And I click on the filter button
+    # Then I should see a "rpm_verify_permissions" link
```

Comment:
Well, more than a workaround, it seems removing coverage right?
Who should provide it? I thought was someone from our company?

Reply:
According to Abid it is enough to have openSCAP running and to have a result. What the result looks like is not our concern. So I just commented that part out until we have the correct Rocky 8 profile available upstream.